src/Makefile

```diff
@@ -117,7 +117,8 @@ CC_FLAGS = \
 	-include $(SRCDIR)/common.h \
 	-DREDIS_MODULE_TARGET \
 	-DREDISTIMESERIES_GIT_SHA=\"$(GIT_SHA)\" \
-	-DREDISMODULE_EXPERIMENTAL_API
+	-DREDISMODULE_EXPERIMENTAL_API \
+	-g -ggdb
```

Comment:
Also need to add it to `LD_FLAGS`. It's better to modify relevant `CC_FLAGS` and `LD_FLAGS` down below (extract the `CC_FLAGS += -g -gdb` and `LD_FLAGS += ...` from the `ifeq ($(DEBUG),1)` block, keeping the `-O0 -DDEBUG -D_DEBUG` in the block.